xdsl/transforms/experimental/ConvertStencilToLLMLIR.py

```diff
@@ -33,6 +33,7 @@ def GetMemRefFromFieldWithLBAndUB(memref_element_type: _TypeElement,
     # lb and ub defines the minimum and maximum coordinates of the resulting memref,
     # so its shape is simply ub - lb, computed here.
     dims = IndexAttr.size_from_bounds(lb, ub)
+    dims.reverse()
```

Comment:
Will it work exactly like this for stencils with dimensions other than 3 as well? 
I am not sure if the swapping is strictly for 1st and 3rd elements or are we supposed to reverse the whole dimension list?